<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1347</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>427</width>
          <height>337</height>
         </rect>
        </property>
        <widget class="QTextEdit" name="textEdit">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>40</y>
           <width>291</width>
           <height>121</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="lockButton">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>180</y>
           <width>83</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Lock</string>
         </property>
        </widget>
        <widget class="QPushButton" name="unlockButton">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>180</y>
           <width>83</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Unlock</string>
         </property>
        </widget>
        <widget class="QLabel" name="modeLabel">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>10</y>
           <width>141</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(233, 185, 110);</string>
         </property>
         <property name="text">
          <string>EDITING</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="saveButton">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>220</y>
           <width>83</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
        <widget class="QPushButton" name="loadButton">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>220</y>
           <width>83</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
        <widget class="QPushButton" name="getAnimalsButton">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>260</y>
           <width>83</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>GetAnimals</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>361</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1347</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
